packages/x-adapter-platform/src/endpoint-adapters/experience-controls.endpoint-adapter.ts

```diff
@@ -0,0 +1,27 @@
+import { endpointAdapterFactory } from '@empathyco/x-adapter';
+// eslint-disable-next-line max-len
+import { ExperienceControlsRequest } from '@empathyco/x-types';
+
+/**.
+ * Default adapter for the experience controls endpoint.
+ *
+ * TODO: change RequestMapper and ResponseMapper
+ *
+ * @public
+ */
+export const experienceControlsEndpointAdapter = endpointAdapterFactory<
+  ExperienceControlsRequest,
+  any
+>({
+  endpoint:
+    'https://config-service.internal.test.empathy.co/public/configs' +
+    '?service=xcontrols&instance={extraParams.instance}',
+  requestMapper: undefined,
+  responseMapper: undefined,
+  defaultRequestOptions: {
+    id: 'experience-controls',
+    parameters: {
+      internal: true
+    }
```

Comment:
These are the static parameters that will be set in the URL. The endpoint for the experience controls doesn't use the `internal` parameter, but the `service` one is a good candidate to set here.

```suggestion
    parameters: {
      service: 'xcontrols'
    }
```